File: C++/Concatenation/Get two numbers as integer input and concatenate them as string.cpp
#include <iostream>
using namespace std;

int main(){
    int num1, num2;
    cout << "Enter the first number: ";
    cin >> num1;
    cout << "Enter the second number: ";
    cin >> num2;

    cout << "The result is = " + to_string(num1) + to_string(num2);

    /*
     * This print also can be done with this following way
     * ---------------------------------------------------
     * string result = to_string(num1) + to_string(num2);
     * cout << "The result is = " + result;
     */
}

File: Java/Concatenation/Get two numbers as integer input and concatenate them as string.java
import java.util.Scanner; 
class numberInputConcatenate {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter the first number: ");
        int num1 = input.nextInt();
        System.out.print("Enter the second number: ");
        int num2 = input.nextInt();

        System.out.println("The result is = " + num1 + num2);

        /*
         * This print also can be done with this following way
         * ---------------------------------------------------
         * String result = "" +num1 + num2;
         * System.out.println( "The result is = " + result );
         */
    }
}